src/readStateExt.m

```diff
@@ -1,11 +1,10 @@
-
 % Copyright (C) 2017 Istituto Italiano di Tecnologia (IIT)
 % All rights reserved.
 %
 % This software may be modified and distributed under the terms of the
 % GNU Lesser General Public License v2.1 or any later version.
 
-function [q, dq, d2q, time] = readStateExt(n, filename)
+function [q, dq, d2q, m, dm, d2m, time] = readStateExt(n, filename)
```

Comment:
For backward compatibility with the existing code probably it is better to add this new outputs **after** time, what do you think @claudia-lat ?

Reply:
@traversaro yes, you're right.